upstream-community-operators/postgresql/postgres-operator.v4.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,690 @@
+    capabilities: Full Lifecycle
+    certified: "false"
+    containerImage: crunchydata/postgres-operator:centos7-4.0.1
+    createdAt: 07/04/2019
+    description: A Postgres Operator from Crunchydata.com
+    repository: https://github.com/CrunchyData/postgres-operator
+    support: crunchydata.com
+  name: postgresoperator.v4.0.1
+  namespace: pgo
+spec:
+  apiservicedefinitions: {}
+  displayName: Crunchy PostgreSQL Enterprise
+  description: |
+    The PostgreSQL Operator runs within a Kubernetes cluster and provides a means to deploy and manage PostgreSQL clusters.
+
+    ## Before You Begin
+
+    There are several manual steps that the cluster administrator must perform prior to installing the operator. The
+    operator must be provided with an initial configuration to run in the cluster, as well as certificates and
+    credentials that need to be generated.
+
+    Start by cloning the operator repository locally.
+
+    ```
+    git clone -b 4.0.1 https://github.com/CrunchyData/postgres-operator.git
+    cd postgres-operator
+    ```
+
+    ### Operator Configuration
+
+    Edit `conf/postgres-operator/pgo.yaml` to configure the operator deployment. Look over all of the options and make any
+    changes necessary for your environment.
+
+    #### Image
+
+    Update the `CCPImageTag` tag to configure the postgres image being used, updating for the version of postgres as needed.
+
+    ```
+    CCPImageTag:  centos7-11.3-2.4.1
+    ```
+
+    #### Storage
+
+    Configure the backend storage for the Persistent Volumes used by each cluster. Depending on the type of persistent
+    storage you wish to make available, adjust the `StorageClass` as necessary. In this case, we are deployed on AWS using `gp2`
+    is the default for the cluster.
+
+    ```
+    storageos:
+      AccessMode:  ReadWriteOnce
+      Size:  1G
+      StorageType:  dynamic
+      StorageClass:  gp2
+      Fsgroup:  26
+    ```
+
+    Once the storage backend is defined, enable the new storage option as needed.
+
+    ```
+    PrimaryStorage: storageos
+    XlogStorage: storageos
+    BackupStorage: storageos
+    ReplicaStorage: storageos
+    BackrestStorage: storageos
+    ```
+
+    ### Certificates
+
+    You will need to either generate new TLS certificates or use existing certificates for the operator API.
+
+    You can generate new self-signed certificates using scripts in the operator repository.
+
+    ```
+    export PGOROOT=$(pwd)
+    cd $PGOROOT/deploy
+    $PGOROOT/deploy/gen-api-keys.sh
+    $PGOROOT/deploy/gen-sshd-keys.sh
+    cd $PGOROOT
+    ```
+
+    ### Configuration and Secrets
+
+    Once the configuration changes have been updated and certificates are in place, we can save the information to the cluster.
+
+    Create the pgo namespace if it does not exist already. This single namespace is where the operator should be deployed to. Postgres clusters will also be deployed here.
+    
+    ```
+    oc create namespace pgo 
```

Comment:
An upstream operator should not reference openshift-specific artifacts such as the ```oc``` command.

Reply:
Yes, you are correct - a documentation oversight on my part. Using 'kubectl' in place of 'oc' will work fine.